roles/figgy/molecule/default/verify.yml

```diff
@@ -21,9 +21,3 @@
     failed_when:
       - not figgy_mount.stat.exists
       - not figgy_mount.stat.isreg
-  - name: test for cogeo-mosaic command
```

Comment:
Purely out of curiosity - what was this for?

Reply:
I am not sure. Running python in a container proved monumentally difficult for me.